tably.py

```diff
@@ -156,6 +159,12 @@ def create_table(self, file):
             return '\n'.join((header, content, footer))
         else:
             return content
+    
+    def create_row(self, line, indent):
+        """Creates a row based on `line` content"""
+        return r'{indent}{indent}{content} \\'.format(
+             indent=indent,
+             content=' & '.join(self.tex_str(line)))
```

Comment:
I agree with you — much better/cleaner than the previous version of `create_row`!